<html><head><title>Results</title><style type="text/css">

* {
  font-family: sans-serif;
}
table {
  border: 1px solid #1C6EA4;
  background-color: #EEEEEE;
  text-align: left;
  border-collapse: collapse;
}
table td, table.s th {
  border: 1px solid #AAAAAA;
  padding: 3px 2px;
  vertical-align: bottom;
}
th {
  vertical-align: bottom;
}
table tr:nth-child(even) {
  background: #D0E4F5;
}
table thead {
  background: #1C6EA4;
  background: -moz-linear-gradient(top, #5592bb 0%, #327cad 66%, #1C6EA4 100%);
  background: -webkit-linear-gradient(top, #5592bb 0%, #327cad 66%, #1C6EA4 100%);
  background: linear-gradient(to bottom, #5592bb 0%, #327cad 66%, #1C6EA4 100%);
  border-bottom: 2px solid #444444;
}
table thead th {
  font-weight: bold;
  color: #FFFFFF;
  border-left: 2px solid #D0E4F5;
}
table thead th:first-child {
  border-left: none;
}

table tfoot {
  font-weight: bold;
  color: #FFFFFF;
  background: #D0E4F5;
  background: -moz-linear-gradient(top, #dcebf7 0%, #d4e6f6 66%, #D0E4F5 100%);
  background: -webkit-linear-gradient(top, #dcebf7 0%, #d4e6f6 66%, #D0E4F5 100%);
  background: linear-gradient(to bottom, #dcebf7 0%, #d4e6f6 66%, #D0E4F5 100%);
  border-top: 2px solid #444444;
}
table tfoot .links {
  text-align: right;
}
table tfoot .links a{
  display: inline-block;
  background: #1C6EA4;
  color: #FFFFFF;
  padding: 2px 8px;
  border-radius: 5px;
}

th  {
    padding-left: 1em;
    padding-right: 1em;
}

table.sortable th:not(.sorttable_sorted):not(.sorttable_sorted_reverse):not(.sorttable_nosort):after {
    content: "â–´â–¾"
}

</style>
<script src="sorttable.js"></script>
</head><body><table class="sortable">
<tr><th>Project</th><th>Total&nbsp;score</th>
<th>coronavirus</th>
<th>covid</th>
<th>covid&nbsp;19</th>
<th>covid-19</th>
<th>covid19</th>
<th>epidemic</th>
<th>epidemiology</th>
<th>flatten&nbsp;the&nbsp;curve</th>
<th>immunology</th>
<th>mers</th>
<th>novel&nbsp;coronavirus</th>
<th>pandemic</th>
<th>sars</th>
<th>sars&nbsp;cov&nbsp;2</th>
<th>sars-cov-2</th>
<th>wuhan</th>
</tr>
<tr><td><b><a href='https://github.com/CSSEGISandData/COVID-19'>COVID-19</a>:</b>&nbsp;Novel Coronavirus (COVID-19) Cases, provided by JHU CSSE
</td><td>50.2</td><td>10.0</td><td>10.0</td><td>10.0</td><td>10.0</td><td></td><td></td><td></td><td></td><td></td><td></td><td>10.0</td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/tokyo-metropolitan-gov/covid19'>covid19</a>:</b>&nbsp;æ±äº¬éƒ½ æ–°å‹ã‚³ãƒ­ãƒŠã‚¦ã‚¤ãƒ«ã‚¹æ„ŸæŸ“ç—‡å¯¾ç­–ã‚µã‚¤ãƒˆ / Tokyo COVID-19 Task Force website
</td><td>35.4</td><td></td><td>8.8</td><td>8.8</td><td>8.8</td><td>8.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/globalcitizen/2019-wuhan-coronavirus-data'>2019-wuhan-coronavirus-data</a>:</b>&nbsp;2019 Wuhan Coronavirus data (COVID-19 / 2019-nCoV)
</td><td>33.4</td><td>6.7</td><td></td><td>6.7</td><td>6.7</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>6.7</td><td></td><td></td><td></td><td>6.7</td></tr>
<tr><td><b><a href='https://github.com/midas-network/COVID-19'>COVID-19</a>:</b>&nbsp;2019 novel coronavirus repository
</td><td>32.3</td><td>6.5</td><td>6.5</td><td>6.5</td><td>6.5</td><td></td><td></td><td></td><td></td><td></td><td></td><td>6.5</td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/pomber/covid19'>covid19</a>:</b>&nbsp;JSON time-series of coronavirus cases (confirmed, deaths and recovered) per country - updated daily
</td><td>29.9</td><td>6.0</td><td>6.0</td><td>6.0</td><td>6.0</td><td>6.0</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/JohnCoene/coronavirus'>coronavirus</a>:</b>&nbsp;ğŸ¦  Novel Coronavirus (COVID-19) Tracker
</td><td>28.7</td><td>5.7</td><td>5.7</td><td>5.7</td><td>5.7</td><td></td><td></td><td></td><td></td><td></td><td></td><td>5.7</td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/datasets/covid-19'>covid-19</a>:</b>&nbsp;Novel Coronavirus 2019 time series data on cases
</td><td>28.7</td><td>5.7</td><td>5.7</td><td>5.7</td><td>5.7</td><td></td><td></td><td></td><td></td><td></td><td></td><td>5.7</td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/FoldingAtHome/coronavirus'>coronavirus</a>:</b>&nbsp;Folding@home COVID-19 efforts
</td><td>27.7</td><td>6.9</td><td>6.9</td><td>6.9</td><td>6.9</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/ExpDev07/coronavirus-tracker-api'>coronavirus-tracker-api</a>:</b>&nbsp;ğŸ¦  A simple and fast (< 200ms) API for tracking the global coronavirus (COVID-19, SARS-CoV-2) outbreak. It's written in python using the ğŸ¼ Flask framework. Supports multiple sources!
</td><td>27.7</td><td>6.9</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>6.9</td><td>6.9</td><td>6.9</td><td></td></tr>
<tr><td><b><a href='https://github.com/kaz-ogiwara/covid19'>covid19</a>:</b>&nbsp;æ–°å‹ã‚³ãƒ­ãƒŠã‚¦ã‚¤ãƒ«ã‚¹ï¼ˆCOVID19ï¼‰ã®å›½å†…ã«ãŠã‘ã‚‹æ„ŸæŸ“ã®çŠ¶æ³ã‚’åšç”ŸåŠ´åƒçœã®å ±é“ç™ºè¡¨è³‡æ–™ã‹ã‚‰ãƒ“ã‚¸ãƒ¥ã‚¢ãƒ«ã«ã¾ã¨ã‚ãŸã€‚
</td><td>27.7</td><td></td><td>6.9</td><td>6.9</td><td>6.9</td><td>6.9</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/neherlab/covid19_scenarios'>covid19_scenarios</a>:</b>&nbsp;Models of COVID-19 outbreak trajectories and hospital demand
</td><td>27.7</td><td></td><td>6.9</td><td>6.9</td><td>6.9</td><td>6.9</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/soroushchehresa/awesome-coronavirus'>awesome-coronavirus</a>:</b>&nbsp;ğŸ¦ Useful projects and resources for COVID-19 (2019 novel Coronavirus)
</td><td>25.8</td><td>6.5</td><td></td><td>6.5</td><td>6.5</td><td></td><td></td><td></td><td></td><td></td><td></td><td>6.5</td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/phildini/stayinghomeclub'>stayinghomeclub</a>:</b>&nbsp;A list of all the companies WFH or events changed because of covid-19
</td><td>24.8</td><td></td><td>6.2</td><td>6.2</td><td>6.2</td><td>6.2</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/github/covid19-dashboard'>covid19-dashboard</a>:</b>&nbsp;A site that displays up to date COVID-19 stats, powered by fastpages.
</td><td>24.8</td><td></td><td>6.2</td><td>6.2</td><td>6.2</td><td>6.2</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/pcm-dpc/COVID-19'>COVID-19</a>:</b>&nbsp;COVID-19 Italia - Monitoraggio situazione
</td><td>24.4</td><td></td><td>8.1</td><td>8.1</td><td>8.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/AaronWard/covid-19-analysis'>covid-19-analysis</a>:</b>&nbsp;Covidify - corona virus report generation for python ğŸ¦ 
</td><td>23.9</td><td></td><td>6.0</td><td>6.0</td><td>6.0</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>6.0</td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/BlankerL/DXY-COVID-19-Data'>DXY-COVID-19-Data</a>:</b>&nbsp;2019æ–°å‹å† çŠ¶ç—…æ¯’ç–«æƒ…æ—¶é—´åºåˆ—æ•°æ®ä»“åº“ | COVID-19/2019-nCoV Infection Time Series Data Warehouse
</td><td>22.2</td><td></td><td>7.4</td><td>7.4</td><td>7.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/BlankerL/DXY-COVID-19-Crawler'>DXY-COVID-19-Crawler</a>:</b>&nbsp;2019æ–°å‹å† çŠ¶ç—…æ¯’ç–«æƒ…å®æ—¶çˆ¬è™«åŠAPI | COVID-19/2019-nCoV Realtime Infection Crawler and API
</td><td>22.2</td><td></td><td>7.4</td><td>7.4</td><td>7.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/rizmaulana/kotlin-mvvm-covid19'>kotlin-mvvm-covid19</a>:</b>&nbsp;This repository contains simple COVID19 data monitoring with android stack MVVM, Live Data, Koin, RxJava, RxBinding, Offline first with simple caching, etc
</td><td>22.0</td><td></td><td>5.5</td><td>5.5</td><td>5.5</td><td>5.5</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/cwoomi/cert-covid19'>cert-covid19</a>:</b>&nbsp;Informationssammlung zum Thema CoVid-19
</td><td>22.0</td><td></td><td>5.5</td><td>5.5</td><td>5.5</td><td>5.5</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/stevenliuyi/covid19'>covid19</a>:</b>&nbsp;an interactive, animated COVID-19 coronavirus map to track the outbreak over time by country and by region for selected countries (including China, Italy, Korea, Japan, France, Germany, United States)
</td><td>21.0</td><td></td><td>5.3</td><td>5.3</td><td>5.3</td><td>5.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/NovelCOVID/API'>API</a>:</b>&nbsp;API for Current cases and more stuff about COVID-19 or the Novel Coronavirus Strain
</td><td>20.8</td><td>6.9</td><td></td><td></td><td>6.9</td><td></td><td></td><td></td><td></td><td></td><td></td><td>6.9</td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/someshkar/covid19india-network'>covid19india-network</a>:</b>&nbsp;:microscope: COVID19 India network cluster
</td><td>20.1</td><td></td><td>5.0</td><td>5.0</td><td>5.0</td><td>5.0</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/twiecki/covid19'>covid19</a>:</b>&nbsp;Analyses about the COVID-19 virus
</td><td>20.1</td><td></td><td>5.0</td><td>5.0</td><td>5.0</td><td>5.0</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/mathdroid/covid-19-api'>covid-19-api</a>:</b>&nbsp;COVID-19 global data (from JHU CSSE for now) as-a-service
</td><td>20.1</td><td></td><td>6.7</td><td>6.7</td><td>6.7</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/echen102/COVID-19-TweetIDs'>COVID-19-TweetIDs</a>:</b>&nbsp;The repository contains an ongoing collection of tweets IDs associated with the novel coronavirus COVID-19 (SARS-CoV-2), which commenced on January 28, 2020.
</td><td>20.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>5.0</td><td></td><td>5.0</td><td>5.0</td><td>5.0</td><td></td></tr>
<tr><td><b><a href='https://github.com/2019ncovmemory/nCovMemory'>nCovMemory</a>:</b>&nbsp;2020æ–°å† è‚ºç‚è®°å¿†ï¼šæŠ¥é“ã€éè™šæ„ä¸ä¸ªäººå™è¿°ï¼ˆæŒç»­æ›´æ–°ï¼‰ Memory of 2020 nCoV: Media Coverage, Non-fiction Writings, and Individual Narratives (Continuously updating)
</td><td>18.6</td><td>9.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>9.3</td></tr>
<tr><td><b><a href='https://github.com/covid19india/covid19india-react'>covid19india-react</a>:</b>&nbsp;Source code of main website
</td><td>18.2</td><td></td><td>4.5</td><td>4.5</td><td>4.5</td><td>4.5</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/alsnhll/SEIR_COVID19'>SEIR_COVID19</a>:</b>&nbsp;SEIR model for COVID-19 infection, including different clinical trajectories of infection
</td><td>18.2</td><td></td><td>4.5</td><td>4.5</td><td>4.5</td><td>4.5</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/geohot/corona'>corona</a>:</b>&nbsp;Reverse engineering SARS-CoV-2
</td><td>17.9</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>6.0</td><td>6.0</td><td>6.0</td><td></td></tr>
<tr><td><b><a href='https://github.com/wlyra/covid19'>covid19</a>:</b>&nbsp;SIR code in python to model the spread of COVID-19
</td><td>17.2</td><td></td><td>4.3</td><td>4.3</td><td>4.3</td><td>4.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/COVID19Tracking/website'>website</a>:</b>&nbsp;COVID 19 Tracking Website
</td><td>17.2</td><td></td><td>5.7</td><td>5.7</td><td>5.7</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/elcronos/COVID-19'>COVID-19</a>:</b>&nbsp;COVID-19 Detector from x-rays using Computer Vision and Deep Learning
</td><td>17.2</td><td></td><td>5.7</td><td>5.7</td><td>5.7</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/CDCgov/SARS-CoV-2_Sequencing'>SARS-CoV-2_Sequencing</a>:</b>&nbsp;(missing)
</td><td>15.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>5.0</td><td>5.0</td><td>5.0</td><td></td></tr>
<tr><td><b><a href='https://github.com/ieee8023/covid-chestxray-dataset'>covid-chestxray-dataset</a>:</b>&nbsp;We are building an open database of COVID-19 cases with chest X-ray or CT images.
</td><td>14.3</td><td></td><td>7.2</td><td>7.2</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/nextstrain/ncov'>ncov</a>:</b>&nbsp;Nextstrain build for novel coronavirus (nCoV)
</td><td>13.9</td><td>6.9</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>6.9</td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/gsarti/covid-papers-browser'>covid-papers-browser</a>:</b>&nbsp;Browse Covid-19 & SARS-CoV-2 Scientific Articles with SciBERT-NLI and BioBERT-NLI ğŸ¦  ğŸ“–
</td><td>13.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>4.5</td><td>4.5</td><td>4.5</td><td></td></tr>
<tr><td><b><a href='https://github.com/ahmadawais/corona-cli'>corona-cli</a>:</b>&nbsp;ğŸ¦  Track the Coronavirus disease (COVID-19) or the Novel Coronavirus Strain in the command line. Both worldwide for all countries, for one country, or for the US States. Fast response time (< 100ms).
</td><td>13.4</td><td>6.7</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>6.7</td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/galaxyproject/SARS-CoV-2'>SARS-CoV-2</a>:</b>&nbsp;Initial analysis of COVID-19 data using Galaxy, BioConda and public research infrastructures
</td><td>12.9</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>4.3</td><td>4.3</td><td>4.3</td><td></td></tr>
<tr><td><b><a href='https://github.com/GuangchuangYu/nCov2019'>nCov2019</a>:</b>&nbsp;query stats of infected coronavirus cases
</td><td>11.9</td><td>6.0</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>6.0</td><td></td></tr>
<tr><td><b><a href='https://github.com/mattroconnor/deep_learning_coronavirus_cure'>deep_learning_coronavirus_cure</a>:</b>&nbsp;Using deep learning to generate novel molecules as candidates for binding with coronavirus protease
</td><td>11.5</td><td>5.7</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>5.7</td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/covid19india/CovidCrowd'>CovidCrowd</a>:</b>&nbsp;A crowd-sourcing platform for the Covid-19 Pandemic
</td><td>11.5</td><td></td><td>3.8</td><td>3.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.8</td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/Hikoyu/SARS-CoV-2'>SARS-CoV-2</a>:</b>&nbsp;SARS-CoV-2 genome sequence data analyses
</td><td>10.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.6</td><td>3.6</td><td>3.6</td><td></td></tr>
<tr><td><b><a href='https://github.com/YiranJing/Coronavirus-Epidemic-2019-nCov'>Coronavirus-Epidemic-2019-nCov</a>:</b>&nbsp;ğŸ‘©ğŸ»â€âš•ï¸Covid-19 estimation and forecast using statistical model; æ–°å‹å† çŠ¶ç—…æ¯’æ­¦æ±‰è‚ºç‚ç»Ÿè®¡æ¨¡å‹é¢„æµ‹
</td><td>10.5</td><td>5.3</td><td></td><td></td><td></td><td></td><td>5.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/839Studio/Novel-Coronavirus-Updates'>Novel-Coronavirus-Updates</a>:</b>&nbsp;æ–°å¢ç¡®è¯Šæ–°å‹å† çŠ¶ç—…æ¯’è‚ºç‚ç»Ÿè®¡æ•°æ®ï¼ˆæ¯æ—¥æ›´æ–°ï¼‰
</td><td>10.0</td><td>5.0</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>5.0</td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/localeai/covid19-live-visualization'>covid19-live-visualization</a>:</b>&nbsp;Live visualization of novel corona virus (COVID19) outbreak
</td><td>9.6</td><td></td><td></td><td></td><td></td><td>4.8</td><td></td><td></td><td></td><td></td><td></td><td>4.8</td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/wuhan2020/wuhan2020'>wuhan2020</a>:</b>&nbsp;æ–°å‹å† çŠ¶ç—…æ¯’é˜²ç–«ä¿¡æ¯æ”¶é›†å¹³å°
</td><td>9.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>9.1</td></tr>
<tr><td><b><a href='https://github.com/dsfsi/covid19za'>covid19za</a>:</b>&nbsp;COVID 19 Data and Dashboard for South Africa
</td><td>8.6</td><td></td><td>4.3</td><td></td><td></td><td>4.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/chrism0dwk/wuhan'>wuhan</a>:</b>&nbsp;Modelling of the nCoV-2019 outbreak in Wuhan, China, by Jon Read, Jess Bridgen, and Chris Jewell at Lancaster University.
</td><td>8.6</td><td></td><td></td><td></td><td></td><td></td><td>4.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>4.3</td></tr>
<tr><td><b><a href='https://github.com/BDI-pathogens/covid-19_instant_tracing'>covid-19_instant_tracing</a>:</b>&nbsp;Quantifying dynamics of SARS-CoV-2 transmission suggests that epidemic control and avoidance is feasible through instantaneous digital contact tracing
</td><td>8.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>4.3</td><td>4.3</td><td></td></tr>
<tr><td><b><a href='https://github.com/SaveWuhan/NewsCoverageOnWuhan'>NewsCoverageOnWuhan</a>:</b>&nbsp;Chinese News coverage on Wuhan during the epidemics outbreak
</td><td>8.1</td><td></td><td></td><td></td><td></td><td></td><td>4.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>4.1</td></tr>
<tr><td><b><a href='https://github.com/therealcyberlord/coronavirus_visualization_and_prediction'>coronavirus_visualization_and_prediction</a>:</b>&nbsp;This repository tracks the spread of the novel coronavirus, also known as SARS-CoV-2. It is a contagious respiratory virus that first started in Wuhan in December 2019. On 2/11/2020, the disease is officially named COVID-19 by the World Health Organization.
</td><td>7.9</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.6</td><td></td><td></td><td>2.6</td><td>2.6</td><td></td></tr>
<tr><td><b><a href='https://github.com/acegid/CoV_Sequences'>CoV_Sequences</a>:</b>&nbsp;SARS-CoV-2 genome from Nigeria
</td><td>7.9</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.6</td><td>2.6</td><td>2.6</td><td></td></tr>
<tr><td><b><a href='https://github.com/covid19-eu-zh/covid19-eu-data'>covid19-eu-data</a>:</b>&nbsp;Automated Data Collection: COVID-19/SARS-COV-2 Cases in EU by Country, State/Province/Local Authorities, and Date
</td><td>7.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.8</td><td>3.8</td><td></td></tr>
<tr><td><b><a href='https://github.com/sorxrob/coronavirus-ph-api'>coronavirus-ph-api</a>:</b>&nbsp;ğŸ¦ A simple and fast (< 200ms) api for tracking the coronavirus (COVID-19, SARS-CoV-2) outbreak in the Philippines.
</td><td>7.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.8</td><td>3.8</td><td></td></tr>
<tr><td><b><a href='https://github.com/Pratitya/wuhan2020-timeline'>wuhan2020-timeline</a>:</b>&nbsp;ä»¥ ç¤¾ä¼šå­¦å¹´é‰´æ¨¡å¼ä½“ä¾‹è§„èŒƒåœ°ç»Ÿç¼–è‡ª2019å¹´12æœˆèµ·æ­¦æ±‰æ–°å† è‚ºç‚ç–«æƒ…è¿›å±•çš„æ—¶é—´çº¿ã€‚
</td><td>7.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>7.4</td></tr>
<tr><td><b><a href='https://github.com/itsksaurabh/go-corona'>go-corona</a>:</b>&nbsp;go-corona is a Golang client library for accessing global coronavirus (COVID-19, SARS-CoV-2) outbreak data.
</td><td>7.2</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.6</td><td>3.6</td><td></td></tr>
<tr><td><b><a href='https://github.com/MhdHejazi/CoronaTracker'>CoronaTracker</a>:</b>&nbsp;Coronavirus tracker app for iOS & macOS with maps & charts
</td><td>6.7</td><td>6.7</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/sagarkarira/coronavirus-tracker-cli'>coronavirus-tracker-cli</a>:</b>&nbsp;Track conronavirus cases from command line. curl https://corona-stats.online/
</td><td>6.7</td><td>6.7</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/nthopinion/covid19'>covid19</a>:</b>&nbsp;Code vs COVID-19: Taking down a Pandemic using machines connected globally to extinguish a biological threat of a super virus.
</td><td>6.7</td><td></td><td></td><td></td><td></td><td>3.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.3</td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/dib-lab/khmer'>khmer</a>:</b>&nbsp;In-memory nucleotide sequence k-mer counting, filtering, graph traversal and more
</td><td>6.7</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>6.7</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/GHDDI-AILab/Targeting2019-nCoV'>Targeting2019-nCoV</a>:</b>&nbsp;Information sharing portal about nCov/SARS/MERS for drug discovery
</td><td>6.7</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.3</td><td></td><td></td><td>3.3</td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/je-suis-tm/quant-trading'>quant-trading</a>:</b>&nbsp;Python quantitative trading strategies including MACD, Pair Trading, Heikin-Ashi, London Breakout, Awesome, Dual Thrust, Parabolic SAR, Bollinger Bands, RSI, Pattern Recognition, CTA, Monte Carlo, Options Straddle
</td><td>6.7</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>6.7</td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/imtsuki/bupt-ncov-report-action'>bupt-ncov-report-action</a>:</b>&nbsp;ğŸ¦  ä½¿ç”¨ GitHub Actions è‡ªåŠ¨å¡«æŠ¥åŒ—é‚® SARS-CoV-2 ç–«æƒ…ä¿¡æ¯
</td><td>6.7</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.3</td><td>3.3</td><td></td></tr>
<tr><td><b><a href='https://github.com/clockworksoul/smudge'>smudge</a>:</b>&nbsp;A lightweight library that provides group member discovery, status dissemination, and failure detection using the SWIM epidemic protocol.
</td><td>6.5</td><td></td><td></td><td></td><td></td><td></td><td>6.5</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/gregoiredavid/france-geojson'>france-geojson</a>:</b>&nbsp;Contours des rÃ©gions, dÃ©partements, arrondissements, cantons et communes de France (mÃ©tropole et dÃ©partements d'outre-mer) au format GeoJSON
</td><td>6.2</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>6.2</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/Kamaropoulos/COVID19Py'>COVID19Py</a>:</b>&nbsp;A tiny Python package for easy access to up-to-date Coronavirus (COVID-19, SARS-CoV-2) cases data.
</td><td>6.2</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.1</td><td>3.1</td><td></td></tr>
<tr><td><b><a href='https://github.com/alexander-stage/athena'>athena</a>:</b>&nbsp;United, we defeat the SARS-CoV-2 virus
</td><td>6.2</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.1</td><td>3.1</td><td></td></tr>
<tr><td><b><a href='https://github.com/parthdmaniar/coronavirus-covid-19-SARS-CoV-2-IoCs'>coronavirus-covid-19-SARS-CoV-2-IoCs</a>:</b>&nbsp;All the IOC's I have gathered which are used directly in coronavirus / covid-19 / SARS-CoV-2 cyber attack campaigns
</td><td>6.2</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.1</td><td>3.1</td><td></td></tr>
<tr><td><b><a href='https://github.com/jspears/mers'>mers</a>:</b>&nbsp;Mongoose Express Rest Service
</td><td>6.0</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>6.0</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/gmarcais/Jellyfish'>Jellyfish</a>:</b>&nbsp;A fast multi-threaded k-mer counter
</td><td>5.7</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>5.7</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/mquad/sars_tutorial'>sars_tutorial</a>:</b>&nbsp;Repository for the tutorial on Sequence-Aware Recommender Systems held at TheWebConf 2019 and ACM RecSys 2018
</td><td>5.7</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>5.7</td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/RadarCODE/awesome-sar'>awesome-sar</a>:</b>&nbsp;A curated list of awesome Synthetic Aperture Radar (SAR) software, libraries, and resources.
</td><td>5.7</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>5.7</td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/ailabstw/COVID19-taiwan'>COVID19-taiwan</a>:</b>&nbsp;Release COVID-19 (SARS-CoV-2) FDA / NHI drugs screening results.
</td><td>5.7</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.9</td><td>2.9</td><td></td></tr>
<tr><td><b><a href='https://github.com/itbirds/2017_wuhan_house_experience'>2017_wuhan_house_experience</a>:</b>&nbsp;2017å¹´æ­¦æ±‰ä¹°æˆ¿ç»å†
</td><td>5.7</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>5.7</td></tr>
<tr><td><b><a href='https://github.com/beoutbreakprepared/nCoV2019'>nCoV2019</a>:</b>&nbsp;Location for summaries and analysis of data related to n-CoV 2019, first reported in Wuhan, China
</td><td>5.7</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>5.7</td></tr>
<tr><td><b><a href='https://github.com/RamiKrispin/coronavirus'>coronavirus</a>:</b>&nbsp;The coronavirus dataset
</td><td>5.5</td><td>5.5</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/mothur/mothur'>mothur</a>:</b>&nbsp;Welcome to the mothur project, initiated by Dr. Patrick Schloss and his software development team in the Department of Microbiology & Immunology at The University of Michigan. This project seeks to develop a single piece of open-source, expandable software to fill the bioinformatics needs of the microbial ecology community.
</td><td>5.5</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>5.5</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/johntruckenbrodt/pyroSAR'>pyroSAR</a>:</b>&nbsp;framework for large-scale SAR satellite data processing
</td><td>5.5</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>5.5</td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/lazd/coronadatascraper'>coronadatascraper</a>:</b>&nbsp;COVID-19 Coronavirus data scraped from government and curated data sources.
</td><td>5.3</td><td>5.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/alfianlosari/CoronaVirusTrackerSwiftUI'>CoronaVirusTrackerSwiftUI</a>:</b>&nbsp;Corona Virus Tracker & Advices iOS App with SwiftUI
</td><td>5.3</td><td>5.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/ondata/covid19italia'>covid19italia</a>:</b>&nbsp;(missing)
</td><td>5.3</td><td></td><td></td><td></td><td></td><td>5.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/opencovid19-fr/data'>data</a>:</b>&nbsp;Consolidation des donnÃ©es de sources officielles concernant l'Ã©pidÃ©mie de COVID19
</td><td>5.3</td><td></td><td></td><td></td><td></td><td>5.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/openZH/covid_19'>covid_19</a>:</b>&nbsp;COVID19 case numbers Cantons of Switzerland and Principality of Liechtenstein (FL) - case numbers include persons tested in the respective area. The data is updated at best once a day (times of collection and update may vary). Start with the README! Help closing the gaps:
</td><td>5.3</td><td></td><td></td><td></td><td>5.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/helium/plumtree'>plumtree</a>:</b>&nbsp;Epidemic Broadcast Trees
</td><td>5.3</td><td></td><td></td><td></td><td></td><td></td><td>5.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/ncovis/choropleth'>choropleth</a>:</b>&nbsp;Choropleth Map of the 2019 Novel Coronavirus Outbreak.
</td><td>5.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>5.3</td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/ZeLianWen/Image-Registration'>Image-Registration</a>:</b>&nbsp;Image registration algorithm. Includes SIFT, SAR-SIFT,PSO-SIFT.
</td><td>5.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>5.3</td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/wuhan-support/wuhan.support'>wuhan.support</a>:</b>&nbsp;#æ­¦æ±‰åŠ æ²¹ #wuhan-support æˆ‘ä»¬å…³æ³¨çš„æ˜¯ä½ æ‰€å…³æ³¨çš„ï¼šé¢å‘ç–«åŒºå†…å¤–æ°‘ä¼—å’ŒåŒ»ç–—æœºæ„çš„å¤šç»´åº¦ä¿¡æ¯æ•´åˆå¹³å°
</td><td>5.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>5.3</td></tr>
<tr><td><b><a href='https://github.com/canghailan/Wuhan-2019-nCoV'>Wuhan-2019-nCoV</a>:</b>&nbsp;2019-nCoV æ–°å† çŠ¶ç—…æ¯’ 2019-12-01è‡³ä»Šå›½å®¶ã€çœã€å¸‚ä¸‰çº§æ¯æ—¥ç»Ÿè®¡æ•°æ®ï¼ˆæ”¯æŒæ¥å£è¯»å–ï¼‰
</td><td>5.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>5.3</td></tr>
<tr><td><b><a href='https://github.com/daenuprobst/covid19-cases-switzerland'>covid19-cases-switzerland</a>:</b>&nbsp;Data from BAG Tweets made useful.
</td><td>5.0</td><td></td><td></td><td></td><td></td><td>5.0</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/celiavelmar/open-covid19-test'>open-covid19-test</a>:</b>&nbsp;An open source COVID-19 autoevaluation test that gives you the same results as coronamadrid.com but does not store your valuable data.
</td><td>5.0</td><td></td><td></td><td></td><td></td><td>5.0</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/pennsignals/chime'>chime</a>:</b>&nbsp;COVID-19 Hospital Impact Model for Epidemics
</td><td>5.0</td><td></td><td></td><td></td><td></td><td></td><td>5.0</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/milaboratory/mixcr'>mixcr</a>:</b>&nbsp;MiXCR is a universal software for fast and accurate extraction of T- and B- cell receptor repertoires from any type of sequencing data. Free for academic use only.
</td><td>5.0</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>5.0</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/TGAC/KAT'>KAT</a>:</b>&nbsp;The K-mer Analysis Toolkit (KAT) contains a number of tools that analyse and compare K-mer spectra.
</td><td>5.0</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>5.0</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/jakobzhao/virus'>virus</a>:</b>&nbsp;Noval Coronavirus Infection Map
</td><td>4.8</td><td>4.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/avischiffmann/Coronavirus-Dashboard'>Coronavirus-Dashboard</a>:</b>&nbsp;This is a website I made to track the ongoing Coronavirus (nCov2019)
</td><td>4.8</td><td>4.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/dssg-pt/covid19pt-data'>covid19pt-data</a>:</b>&nbsp;ğŸ˜·ï¸ğŸ‡µğŸ‡¹ Dados relativos Ã  pandemia COVID-19 em Portugal
</td><td>4.8</td><td></td><td></td><td></td><td></td><td>4.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/reustle/covid19japan'>covid19japan</a>:</b>&nbsp;(missing)
</td><td>4.8</td><td></td><td></td><td></td><td></td><td>4.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/turicas/covid19-br'>covid19-br</a>:</b>&nbsp;Dados diÃ¡rios mais recentes do coronavÃ­rus por municÃ­pio brasileiro
</td><td>4.8</td><td></td><td></td><td></td><td></td><td>4.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/dominictarr/epidemic-broadcast-trees'>epidemic-broadcast-trees</a>:</b>&nbsp;bandwidth efficient broadcast gossip
</td><td>4.8</td><td></td><td></td><td></td><td></td><td></td><td>4.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/reconhub/epitrix'>epitrix</a>:</b>&nbsp;Small Helpers and Tricks for Epidemics Analysis
</td><td>4.8</td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/cmrivers/epipy'>epipy</a>:</b>&nbsp;Python tools for epidemiology
</td><td>4.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td>4.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/IFRCGo/cbs'>cbs</a>:</b>&nbsp;Red Cross: Community-Based Surveillance
</td><td>4.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td>4.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/statnet/EpiModel'>EpiModel</a>:</b>&nbsp;Mathematical Modeling of Infectious Disease Dynamics
</td><td>4.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td>4.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/jisungk/RIDDLE'>RIDDLE</a>:</b>&nbsp;Race and ethnicity Imputation from Disease history with Deep LEarning
</td><td>4.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td>4.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/pnpnpn/dna2vec'>dna2vec</a>:</b>&nbsp;dna2vec: Consistent vector representations of variable-length k-mers
</td><td>4.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>4.8</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/refresh-bio/KMC'>KMC</a>:</b>&nbsp;Fast and frugal disk based k-mer counter
</td><td>4.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>4.8</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/fbreitwieser/krakenuniq'>krakenuniq</a>:</b>&nbsp;ğŸ™ KrakenUniq: Metagenomics classifier with unique k-mer counting for more specific results
</td><td>4.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>4.8</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/yjlou/2019-nCov'>2019-nCov</a>:</b>&nbsp;Use Google Maps Timeline data to compare with COVID-19 patient history location.
</td><td>4.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>4.8</td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/wangpengnorman/SAR-Strong-Baseline-for-Text-Recognition'>SAR-Strong-Baseline-for-Text-Recognition</a>:</b>&nbsp;SAR: A Simple and Strong Baseline for Irregular Text Recognition
</td><td>4.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>4.8</td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/bioinf-mcb/gisaid-scrapper'>gisaid-scrapper</a>:</b>&nbsp;Scrapping tool for GISAID data regarding SARS-CoV-2
</td><td>4.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td>2.4</td><td></td></tr>
<tr><td><b><a href='https://github.com/pwr22/covbot'>covbot</a>:</b>&nbsp;Matrix bot tracking SARS-COV-2 statistics.
</td><td>4.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td>2.4</td><td></td></tr>
<tr><td><b><a href='https://github.com/egonw/SARS-CoV-2-Queries'>SARS-CoV-2-Queries</a>:</b>&nbsp;Wikidata Queries around the SARS-CoV-2 virus and pandemic
</td><td>4.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td>2.4</td><td></td></tr>
<tr><td><b><a href='https://github.com/mew/Coronavirus'>Coronavirus</a>:</b>&nbsp;Java API Wrapper for tracking coronavirus (COVID-19, SARS-CoV-2) via https://git.io/Jvoep
</td><td>4.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td>2.4</td><td></td></tr>
<tr><td><b><a href='https://github.com/NeoHuang/corona-weg'>corona-weg</a>:</b>&nbsp;SARS-CoV-2 germany dashboard and API
</td><td>4.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td>2.4</td><td></td></tr>
<tr><td><b><a href='https://github.com/feiglab/sars-cov-2-proteins'>sars-cov-2-proteins</a>:</b>&nbsp;(missing)
</td><td>4.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td>2.4</td><td></td></tr>
<tr><td><b><a href='https://github.com/dlchao/corvid'>corvid</a>:</b>&nbsp;Corvid, an agent-based SARS-CoV-2 (COVID 2019) transmission model
</td><td>4.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td>2.4</td><td></td></tr>
<tr><td><b><a href='https://github.com/vsaase/covid19selfreport'>covid19selfreport</a>:</b>&nbsp;Providing a dashboard for geographically self reporting symptomatic cases of Corona Virus Disease (Covid-19, nCov19, Coronavirus SARS-CoV-2), using Flask and Leaflet and Google App Engine.
</td><td>4.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td>2.4</td><td></td></tr>
<tr><td><b><a href='https://github.com/katarinabraun/SARS-CoV-2_sequencing'>SARS-CoV-2_sequencing</a>:</b>&nbsp;SARS-CoV-2 Deep Sequencing
</td><td>4.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td>2.4</td><td></td></tr>
<tr><td><b><a href='https://github.com/jsantell/SARS-CoV-2-genome'>SARS-CoV-2-genome</a>:</b>&nbsp;SARS-CoV-2 genome
</td><td>4.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td>2.4</td><td></td></tr>
<tr><td><b><a href='https://github.com/CloseChoice/FlatCurver'>FlatCurver</a>:</b>&nbsp;Repo for Hackathon WevsVirus. The aim is to visualize effects of measures aimed to containing the SARS-CoV-2 virus.
</td><td>4.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td>2.4</td><td></td></tr>
<tr><td><b><a href='https://github.com/jianxu305/nCov2019_analysis'>nCov2019_analysis</a>:</b>&nbsp;Analysis of 2019-nCov coronavirus data
</td><td>4.5</td><td>4.5</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/coronavirus-diary/coronavirus-diary'>coronavirus-diary</a>:</b>&nbsp;Ethically sourcing data for the fight against disease.
</td><td>4.5</td><td>4.5</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/midudev/covid-19-spread-simulator'>covid-19-spread-simulator</a>:</b>&nbsp;COVID-19 (Coronavirus) Spread Simulator ğŸ¦ 
</td><td>4.5</td><td>4.5</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/chschoenenberger/covid19_dashboard'>covid19_dashboard</a>:</b>&nbsp;This is a dashboard, similar to the one from Johns Hopkins usind open source technology.
</td><td>4.5</td><td></td><td></td><td></td><td></td><td>4.5</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/antigenomics/vdjdb-db'>vdjdb-db</a>:</b>&nbsp;Git-based TCR database storage & management. Submissions welcome!
</td><td>4.5</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>4.5</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/boo1ean/mersenne-twister'>mersenne-twister</a>:</b>&nbsp;Mersenne Twister pseudorandom number generator
</td><td>4.5</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>4.5</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/dm6718/RITSAR'>RITSAR</a>:</b>&nbsp;Synthetic Aperture Radar (SAR) Image Processing Toolbox for Python
</td><td>4.5</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>4.5</td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/birgander2/PyRAT'>PyRAT</a>:</b>&nbsp;General purpose Synthetic Aperture Radar (SAR) postprocessing software package
</td><td>4.5</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>4.5</td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/wuhan2020/wuhan2020.github.io'>wuhan2020.github.io</a>:</b>&nbsp;Wuhan2020 Community Website
</td><td>4.5</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>4.5</td></tr>
<tr><td><b><a href='https://github.com/konradkalemba/korona.ws'>korona.ws</a>:</b>&nbsp;ğŸ—º Coronavirus interactive map of Poland
</td><td>4.3</td><td>4.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/emergenzeHack/covid19italia'>covid19italia</a>:</b>&nbsp;Condividiamo informazioni e segnalazioni sul COVID19
</td><td>4.3</td><td></td><td></td><td></td><td></td><td>4.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/mikessh/vdjtools'>vdjtools</a>:</b>&nbsp;Post-analysis of immune repertoire sequencing data
</td><td>4.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>4.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/cslarsen/mersenne-twister'>mersenne-twister</a>:</b>&nbsp;This Mersenne Twister is a fast pseudo-random number generator (PRNG) in C++
</td><td>4.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>4.3</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/pigulla/mersennetwister'>mersennetwister</a>:</b>&nbsp;A standalone, pure JavaScript implementation of the Mersenne Twister pseudo random number generator. Compatible with Node.js, requirejs and browser environments. Packages are available for npm, Jam and Bower.
</td><td>4.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>4.3</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/splatlab/squeakr'>squeakr</a>:</b>&nbsp;Squeakr: An Exact and Approximate k -mer Counting System
</td><td>4.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>4.3</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/RolfRolles/PandemicML'>PandemicML</a>:</b>&nbsp;Pandemic binary program analysis framework in OCaml (abandoned)
</td><td>4.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>4.3</td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/lachlanjc/covid19'>covid19</a>:</b>&nbsp;Visualize & track the 2020 COVID-19 pandemic by country.
</td><td>4.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>4.3</td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/JohnSundell/IndieSupportWeeks'>IndieSupportWeeks</a>:</b>&nbsp;A two-week effort to help support indie developers shipping apps on Apple's platforms who have been financially impacted by the COVID-19 pandemic.
</td><td>4.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>4.3</td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/cTn-dev/Phoenix-FlightController'>Phoenix-FlightController</a>:</b>&nbsp;Flight Controller designed for SAR operations
</td><td>4.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>4.3</td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/blab/sars-like-cov'>sars-like-cov</a>:</b>&nbsp;This is a one-off Nextstrain build for SARS-like coronaviruses
</td><td>4.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>4.3</td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/Rank23/COVID19'>COVID19</a>:</b>&nbsp;Using Kalman Filter to Predict Corona Virus Spread
</td><td>4.1</td><td></td><td></td><td></td><td></td><td>4.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/lifen-labs/covid'>covid</a>:</b>&nbsp;Covid-19 patients' self-monitoring at home via forms sent by SMS
</td><td>4.1</td><td></td><td>4.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/gabgoh/epcalc'>epcalc</a>:</b>&nbsp;Epidemic Calculator
</td><td>4.1</td><td></td><td></td><td></td><td></td><td></td><td>4.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/springer-math/Mathematics-of-Epidemics-on-Networks'>Mathematics-of-Epidemics-on-Networks</a>:</b>&nbsp;Source code accompanying 'Mathematics of Epidemics on Networks' by Kiss, Miller, and Simon http://www.springer.com/us/book/9783319508047 . Documentation for the software package is at https://epidemicsonnetworks.readthedocs.io/en/latest/
</td><td>4.1</td><td></td><td></td><td></td><td></td><td></td><td>4.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/najibelmokhtari/basic-epidemic-simulation'>basic-epidemic-simulation</a>:</b>&nbsp;A very basic epidemic simulation for educational purposes. This code was made as a part of a YouTube episode on a pop science channel hosted by Najib El Mokhtari. Any enhancements are welcome and unrestricted.
</td><td>4.1</td><td></td><td></td><td></td><td></td><td></td><td>4.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/pzivich/zEpid'>zEpid</a>:</b>&nbsp;Epidemiology analysis package
</td><td>4.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td>4.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/pzivich/Python-for-Epidemiologists'>Python-for-Epidemiologists</a>:</b>&nbsp;Tutorial in Python targeted at Epidemiologists. Will discuss the basics of analysis in Python 3
</td><td>4.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td>4.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/InstituteforDiseaseModeling/EMOD'>EMOD</a>:</b>&nbsp;Source files for building the IDM EMOD disease transmission model.
</td><td>4.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td>4.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/bioinformatics-centre/BayesTyper'>BayesTyper</a>:</b>&nbsp;A method for variant graph genotyping based on exact alignment of k-mers
</td><td>4.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>4.1</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/the-robot/covid19-updates'>covid19-updates</a>:</b>&nbsp;Monitoring service and bot to give updates about COVID-19 (Novel Coronavirus)
</td><td>4.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>4.1</td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/tmacdou4/2019-nCov'>2019-nCov</a>:</b>&nbsp;Efforts towards proposing a potentially highly active molecule against a target protein of the 2019 Novel Coronavirus
</td><td>4.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>4.1</td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/lestweforget/wuhan2019'>wuhan2019</a>:</b>&nbsp;æœ¬é¡¹ç›®ä¸»è¦å°±ä¸­å›½å›½å†…éƒ¨åˆ†ä¸»æµåª’ä½“å¯¹æ–°å† è‚ºç‚ï¼ˆ2019.12-ï¼‰çš„æŠ¥é“è¿›è¡Œå¤‡ä»½ã€å­˜æ¡£ã€‚åŒ…æ‹¬è¢«å½“å±€åˆ é™¤åœ¨å†…çš„æ‰€æœ‰æ–‡ç« ä»¥ æ—¥æœŸ+æ ‡é¢˜ çš„æ ¼å¼ç”ŸæˆPDFæ–‡ä»¶ï¼Œä¾›è¯»è€…æŸ¥é˜…ã€‚æ–‡ç« ç‰ˆæƒå½’ä½œè€…åŠå…¶æ‰€å±æœºæ„æ‰€æœ‰ã€‚
</td><td>4.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>4.1</td></tr>
<tr><td><b><a href='https://github.com/jugetaozi/2019-wuhan-house-exp'>2019-wuhan-house-exp</a>:</b>&nbsp;2019å¹´æ­¦æ±‰ä¹°æˆ¿ç»å† ---- æŒç»­æ›´æ–° wuhan_house
</td><td>4.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>4.1</td></tr>
<tr><td><b><a href='https://github.com/theleadio/coronatracker'>coronatracker</a>:</b>&nbsp;Keeps track of the development of Wuhan Corona virus
</td><td>4.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>4.1</td></tr>
<tr><td><b><a href='https://github.com/RamiKrispin/coronavirus_dashboard'>coronavirus_dashboard</a>:</b>&nbsp;The Coronavirus Dashboard
</td><td>3.8</td><td>3.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/plotly/dash-opioid-epidemic-demo'>dash-opioid-epidemic-demo</a>:</b>&nbsp;US county data for poision-induced deaths, years 1999-2015
</td><td>3.8</td><td></td><td></td><td></td><td></td><td></td><td>3.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/CodeForPhilly/chime'>chime</a>:</b>&nbsp;COVID-19 Hospital Impact Model for Epidemics
</td><td>3.8</td><td></td><td></td><td></td><td></td><td></td><td>3.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/bcgsc/LINKS'>LINKS</a>:</b>&nbsp;â›“Long Interval Nucleotide K-mer Scaffolder
</td><td>3.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.8</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/kmyk/mersenne-twister-predictor'>mersenne-twister-predictor</a>:</b>&nbsp;Predict MT19937 PRNG, from preceding 624 generated numbers. There is a specialization for the "random" of Python standard library.
</td><td>3.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.8</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/jandrewrfarrell/RUFUS'>RUFUS</a>:</b>&nbsp;RUFUS k-mer based genomic variant detection
</td><td>3.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.8</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/kennygrant/coronavirus'>coronavirus</a>:</b>&nbsp;Graphing global deaths and confirmed cases due to novel coronavirus (Covid-19)
</td><td>3.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.8</td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/arya/pandemic'>pandemic</a>:</b>&nbsp;"Because information needs to be contagious."
</td><td>3.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.8</td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/hfiguiere/no-flash'>no-flash</a>:</b>&nbsp;A WebExtension to help eradicate the Flash pandemic.
</td><td>3.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.8</td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/hunterlew/mstar_with_machine_learning'>mstar_with_machine_learning</a>:</b>&nbsp;Simple implementation of sar target recognition using machine learning methods
</td><td>3.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.8</td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/antonlukin/2019-nCoV'>2019-nCoV</a>:</b>&nbsp;Wuhan coronavirus outbreak
</td><td>3.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.8</td></tr>
<tr><td><b><a href='https://github.com/wuhancrisis/wuhancrisis.github.io'>wuhancrisis.github.io</a>:</b>&nbsp;æ­¦æ±‰åŒèƒå£ç½©ä¸‹çš„å‘å–Š | Humanitarian Crisis in Wuhan, China
</td><td>3.8</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.8</td></tr>
<tr><td><b><a href='https://github.com/DavideMagno/ItalianCovidData'>ItalianCovidData</a>:</b>&nbsp;Historical data about Covid19 epidemic in Italy
</td><td>3.6</td><td></td><td></td><td></td><td></td><td></td><td>3.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/sseefried/docker-epidemic-build-env'>docker-epidemic-build-env</a>:</b>&nbsp;A Dockerfile to build a development environment for Android version of Epidemic.
</td><td>3.6</td><td></td><td></td><td></td><td></td><td></td><td>3.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/tjhladish/EpiFire'>EpiFire</a>:</b>&nbsp;A fast C++ API for simulating the spread of epidemics on contact networks.
</td><td>3.6</td><td></td><td></td><td></td><td></td><td></td><td>3.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/SwissTPH/openmalaria'>openmalaria</a>:</b>&nbsp;A simulator of malaria epidemiology and control
</td><td>3.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/qmarcou/IGoR'>IGoR</a>:</b>&nbsp;IGoR is a C++ software designed to infer V(D)J recombination related processes from sequencing data. Find full documentation at:
</td><td>3.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/imminfo/tcr'>tcr</a>:</b>&nbsp;[DEPRECATED, see https://immunarch.com/] tcR: an R package for immune receptor repertoire advanced data analysis.
</td><td>3.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/mikessh/migec'>migec</a>:</b>&nbsp;A RepSeq processing swiss-knife
</td><td>3.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/mikessh/migmap'>migmap</a>:</b>&nbsp;HTS-compatible wrapper for IgBlast V-(D)-J mapping tool
</td><td>3.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/immunomind/immunarch'>immunarch</a>:</b>&nbsp;immunarch: An R Package for Painless Bioinformatics Analysis of T-cell and B-cell Immune Repertoire Data
</td><td>3.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/lasigeBioTM/MER'>MER</a>:</b>&nbsp;Minimal Named-Entity Recognizer (MER)
</td><td>3.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.6</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/HopkinsIDD/ncov_incubation'>ncov_incubation</a>:</b>&nbsp;Estimating the incubation time of the novel coronavirus (nCoV-2019) based on traveler data using coarse data tools
</td><td>3.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.6</td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/k1m0ch1/covid-19-api'>covid-19-api</a>:</b>&nbsp;ğŸ° Simple and ğŸ¦… fast covid-19 api global data parser using Flask, data gathered from Novel Coronavirus (COVID-19) Cases, provided by JHU CSSE
</td><td>3.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.6</td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/siaw23/kovid'>kovid</a>:</b>&nbsp;ğŸ¦ A CLI to fetch and compare the 2019 coronavirus pandemic statistics.
</td><td>3.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.6</td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/openforis/opensarkit'>opensarkit</a>:</b>&nbsp;Tools for Automatic Preprocessing of SAR Imagery
</td><td>3.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.6</td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/aloiscochard/sarsi'>sarsi</a>:</b>&nbsp;A universal quickfix toolkit and his protocol.
</td><td>3.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.6</td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/FelixZFB/wuhan_2019-nCoV'>wuhan_2019-nCoV</a>:</b>&nbsp;æ­¦æ±‰2019æ–°å‹å† çŠ¶ç—…æ¯’ç–«æƒ…å¯è§†åŒ–(å…¨å›½ç–«æƒ…åœ°å›¾åŠæ—¶é—´è½´å˜åŒ–)
</td><td>3.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.6</td></tr>
<tr><td><b><a href='https://github.com/jriou/wcov'>wcov</a>:</b>&nbsp;Pattern of early human-to-human transmission of Wuhan 2019-nCoV
</td><td>3.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.6</td></tr>
<tr><td><b><a href='https://github.com/koushikkothagal/coronavirus-tracker'>coronavirus-tracker</a>:</b>&nbsp;Spring Boot Java app to track coronavirus confirmed cases across the globe
</td><td>3.3</td><td>3.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/necsi/covid19'>covid19</a>:</b>&nbsp;This is the repository for all of the code that necsi is writing about covid19, along with documentation.
</td><td>3.3</td><td></td><td></td><td></td><td></td><td>3.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/stop-covid19-hyogo/covid19'>covid19</a>:</b>&nbsp;å…µåº«çœŒéå…¬å¼ æ–°å‹ã‚³ãƒ­ãƒŠã‚¦ã‚¤ãƒ«ã‚¹ã¾ã¨ã‚ã‚µã‚¤ãƒˆ
</td><td>3.3</td><td></td><td></td><td></td><td></td><td>3.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/hifiaz/covid19'>covid19</a>:</b>&nbsp;(missing)
</td><td>3.3</td><td></td><td></td><td></td><td></td><td>3.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/sseefried/open-epidemic-game'>open-epidemic-game</a>:</b>&nbsp;A game about exponential growth written in Haskell
</td><td>3.3</td><td></td><td></td><td></td><td></td><td></td><td>3.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/EpiModel/EpiModel-Gallery'>EpiModel-Gallery</a>:</b>&nbsp;Gallery of Network-Based Epidemic Model Templates for EpiModel
</td><td>3.3</td><td></td><td></td><td></td><td></td><td></td><td>3.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/objornstad/epimdr'>epimdr</a>:</b>&nbsp;R package for "Epidemics: Models and data using R"
</td><td>3.3</td><td></td><td></td><td></td><td></td><td></td><td>3.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/phac-nml/irida'>irida</a>:</b>&nbsp;Canadaâ€™s Integrated Rapid Infectious Disease Analysis Platform for Genomic Epidemiology
</td><td>3.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/reichlab/flusight'>flusight</a>:</b>&nbsp;Influenza forecasts visualizer
</td><td>3.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/mer-packages/libsignon'>libsignon</a>:</b>&nbsp;signond packaging for Mer
</td><td>3.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.3</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/dmt4/sfa-mer'>sfa-mer</a>:</b>&nbsp;Sailfish/Mer image build scripts
</td><td>3.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.3</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/artic-network/artic-ncov2019'>artic-ncov2019</a>:</b>&nbsp;ARTIC nanopore protocol for nCoV2019 novel coronavirus
</td><td>3.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.3</td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/eebrown/data2019nCoV'>data2019nCoV</a>:</b>&nbsp;Data on the 2019 Novel Coronavirus Outbreak (R Package)
</td><td>3.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.3</td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/trvrb/dynamics-practical'>dynamics-practical</a>:</b>&nbsp;Inferring spatiotemporal dynamics of the H1N1 influenza pandemic from sequence data
</td><td>3.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.3</td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/LK-LAB/SARS2-Stat-KR'>SARS2-Stat-KR</a>:</b>&nbsp;ì¤‘êµ­ ìš°í•œ ë°”ì´ëŸ¬ìŠ¤ í•œêµ­ í™•ì§„ì í†µê³„
</td><td>3.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.3</td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/bakerunavco/SSARA'>SSARA</a>:</b>&nbsp;Seamless SAR Archive project repository
</td><td>3.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.3</td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/ppalucha/ksar2'>ksar2</a>:</b>&nbsp;Fork of ksar - a sar grapher
</td><td>3.3</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.3</td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/ihmeuw/dismod_mr'>dismod_mr</a>:</b>&nbsp;An Integrative Metaregression Framework for Descriptive Epidemiology
</td><td>3.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/UMMISCO/kendrick'>kendrick</a>:</b>&nbsp;Domain-Specific Modeling for Epidemiology
</td><td>3.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/Japrin/sscClust'>sscClust</a>:</b>&nbsp;simpler single cell RNAseq data clustering
</td><td>3.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/CRI-iAtlas/shiny-iatlas'>shiny-iatlas</a>:</b>&nbsp;An interactive web portal for exploring immuno-oncology data
</td><td>3.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/RGLab/ImmuneSpaceR'>ImmuneSpaceR</a>:</b>&nbsp;An R Interface to the ImmuneSpace database portal
</td><td>3.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/antigenomics/vdjviz'>vdjviz</a>:</b>&nbsp;A lightweight immune repertoire browser
</td><td>3.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/atifrahman/HAWK'>HAWK</a>:</b>&nbsp;Hitting associations with k-mers
</td><td>3.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.1</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/GATB/dsk'>dsk</a>:</b>&nbsp;k-mer counting software
</td><td>3.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.1</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/docligot/coronatracker-analytics'>coronatracker-analytics</a>:</b>&nbsp;Analysis and reports based on novel coronavirus (nCoV-2019) data.
</td><td>3.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.1</td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/iidx/corona-tracker'>corona-tracker</a>:</b>&nbsp;Convert Novel coronavirus(2019-nCoV) statistical data from the Chinese medical community(dxy.cn) to json format
</td><td>3.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.1</td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/jwcxz/vim-pandemic'>vim-pandemic</a>:</b>&nbsp;Manage remote pathogen bundles from multiple source types
</td><td>3.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.1</td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/AdaCompNUS/sarsop'>sarsop</a>:</b>&nbsp;Efficient Point-Based POMDP Planning by Approximating
</td><td>3.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.1</td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/mbaldessari/sarstats'>sarstats</a>:</b>&nbsp;Creates a PDF report out of sar files
</td><td>3.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.1</td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/Jacksgong/wuhan-2019-nCoV'>wuhan-2019-nCoV</a>:</b>&nbsp;æ­¦æ±‰2019nCovä¿¡æ¯çˆ¬è™«
</td><td>3.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.1</td></tr>
<tr><td><b><a href='https://github.com/wybert/open-wuhan-ncov-illness-data'>open-wuhan-ncov-illness-data</a>:</b>&nbsp;è¿™ä¸ªé¡¹ç›®æœ‰å…³æœ‰å…³ æ­¦æ±‰è‚ºç‚ 2019-ncovçš„ç›¸å…³ç—…ä¾‹æ•°æ®çš„åˆ†äº«ã€‚ä»gitlabè¿ç§»è¿‡æ¥ã€‚å‘å¸ƒé¡µè¿™é‡Œ
</td><td>3.1</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>3.1</td></tr>
<tr><td><b><a href='https://github.com/reconhub/learn'>learn</a>:</b>&nbsp;RECON learn: a free, open platform for training material on epidemics analysis
</td><td>2.9</td><td></td><td></td><td></td><td></td><td></td><td>2.9</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/R4EPI/sitrep'>sitrep</a>:</b>&nbsp;Report templates and helper functions for field epidemiologists
</td><td>2.9</td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.9</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/dmnfarrell/epitopepredict'>epitopepredict</a>:</b>&nbsp;Python package and command line tool for epitope prediction
</td><td>2.9</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.9</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/mfreeborn/coronavirus-tracker'>coronavirus-tracker</a>:</b>&nbsp;Interactive map plotting the spread of novel coronavirus.
</td><td>2.9</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.9</td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/WarenGonzaga/covid19-tracker-cli'>covid19-tracker-cli</a>:</b>&nbsp;Track Novel Coronavirus (COVID-19) or Coronavirus disease cases from command line interface. It Supports terminal for linux and macos, command prompt for windows, and termux for android. Fast response time (~< 100ms). curl https://covid19tracker.xyz
</td><td>2.9</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.9</td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/alexzherdev/pandemic'>pandemic</a>:</b>&nbsp;A Redux-based implementation of the board game Pandemic.
</td><td>2.9</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.9</td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/raffg/covid-19'>covid-19</a>:</b>&nbsp;dashboard to monitor the COVID-19 pandemic
</td><td>2.9</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.9</td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/nf-core/covid19'>covid19</a>:</b>&nbsp;DEVELOPMENT ON HOLD: A pipeline for the analysis of COVID-19 pandemic data.
</td><td>2.9</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.9</td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/databrew/covid19'>covid19</a>:</b>&nbsp;Data visualization related to the COVID-19 pandemic
</td><td>2.9</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.9</td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/binhe22/HackWuhan2015List'>HackWuhan2015List</a>:</b>&nbsp;Hackathon Wuhan 2015 é¡¹ç›®åˆ—è¡¨
</td><td>2.9</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.9</td></tr>
<tr><td><b><a href='https://github.com/eastmountyxz/Wuhan-data-analysis'>Wuhan-data-analysis</a>:</b>&nbsp;è¯¥ç³»åˆ—èµ„æºæ˜¯Pythonç–«æƒ…å¤§æ•°æ®åˆ†æï¼Œæ¶‰åŠç½‘ç»œçˆ¬è™«ã€å¯è§†åŒ–åˆ†æã€GISåœ°å›¾ã€æƒ…æ„Ÿåˆ†æã€èˆ†æƒ…åˆ†æã€ä¸»é¢˜æŒ–æ˜ã€å¨èƒæƒ…æŠ¥æº¯æºã€çŸ¥è¯†å›¾è°±ã€é¢„æµ‹é¢„è­¦åŠAIå’ŒNLPåº”ç”¨ç­‰ã€‚æ¨èå¤§å®¶ç»“åˆä½œè€…CSDNåšå®¢é˜…è¯»ï¼Œæ­¦æ±‰å¿…èƒœã€æ¹–åŒ—å¿…èƒœã€ä¸­å›½å¿…èƒœï¼
</td><td>2.9</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.9</td></tr>
<tr><td><b><a href='https://github.com/reconhub/reconhub.github.io'>reconhub.github.io</a>:</b>&nbsp;Official website for RECON - The R Epidemics CONsortium
</td><td>2.6</td><td></td><td></td><td></td><td></td><td></td><td>2.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/Estelle0217/COVID-19-Epidemic-Dataset'>COVID-19-Epidemic-Dataset</a>:</b>&nbsp;(missing)
</td><td>2.6</td><td></td><td></td><td></td><td></td><td></td><td>2.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/y1han/epidemic-simulation'>epidemic-simulation</a>:</b>&nbsp;A python programme about epidemic simulation
</td><td>2.6</td><td></td><td></td><td></td><td></td><td></td><td>2.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/ssbc/ssb-ebt'>ssb-ebt</a>:</b>&nbsp;secure scuttlebutt replication with epidemic-broadcast-trees
</td><td>2.6</td><td></td><td></td><td></td><td></td><td></td><td>2.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/abhijitmjj/Prediction-of-epidemic-disease-dynamics-using-Machine-learning-model'>Prediction-of-epidemic-disease-dynamics-using-Machine-learning-model</a>:</b>&nbsp;(missing)
</td><td>2.6</td><td></td><td></td><td></td><td></td><td></td><td>2.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/andrejbauer/social-distancing-simulator'>social-distancing-simulator</a>:</b>&nbsp;An artificial simulation of social distancing in the time of an epidemic.
</td><td>2.6</td><td></td><td></td><td></td><td></td><td></td><td>2.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/sdwfrost/mevr-course-2016'>mevr-course-2016</a>:</b>&nbsp;Molecular Epidemiology of Viruses in R
</td><td>2.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/AlertaDengue/AlertaDengue'>AlertaDengue</a>:</b>&nbsp;Portal de dados do Projeto Alerta Dengue
</td><td>2.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/rambaut/MERS-Cases'>MERS-Cases</a>:</b>&nbsp;(missing)
</td><td>2.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.6</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/coronafighter/coronaSEIR'>coronaSEIR</a>:</b>&nbsp;Simple SEIR model Python script for the COVID-19 pandemic with real world data comparison.
</td><td>2.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.6</td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/jquelin/games-pandemic'>games-pandemic</a>:</b>&nbsp;a cooperative pandemic board game written in perl
</td><td>2.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.6</td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/wuhan2020/wuhan2020-frontend-react-native-app'>wuhan2020-frontend-react-native-app</a>:</b>&nbsp;(missing)
</td><td>2.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.6</td></tr>
<tr><td><b><a href='https://github.com/shadow12138/WuHanGanBaDei'>WuHanGanBaDei</a>:</b>&nbsp;æ–°å‹å† çŠ¶ç—…æ¯’ç–«æƒ…å¯è§†åŒ–
</td><td>2.6</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.6</td></tr>
<tr><td><b><a href='https://github.com/dlchao/FluTE'>FluTE</a>:</b>&nbsp;Agent-based influenza epidemic model
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/ccattuto/SIRtoy'>SIRtoy</a>:</b>&nbsp;Javascript implementation of an epidemic SIR model
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/moonlightlong/epidemic'>epidemic</a>:</b>&nbsp;(missing)
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/chgigot/epiphy'>epiphy</a>:</b>&nbsp;Analyzing plant disease epidemics
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/trvrb/antigen'>antigen</a>:</b>&nbsp;Simulating virus evolution and epidemiology
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/ahgroup/DSAIDE'>DSAIDE</a>:</b>&nbsp;R Package - Dynamical Systems Approach to Infectious Disease Epidemiology
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/andersen-lab/zika-epidemiology'>zika-epidemiology</a>:</b>&nbsp;Zika Epidemiology - Case Numbers
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/AABoyles/MicrobeTrace'>MicrobeTrace</a>:</b>&nbsp;The Visualization Multitool for Molecular Epidemiology and Bioinformatics
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/kelseyandersen/PLP6905'>PLP6905</a>:</b>&nbsp;Epidemiology PLP6905
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/dmcalli2/Advanced_epidemiology_course'>Advanced_epidemiology_course</a>:</b>&nbsp;Material for advanced epidemiology module in MPH
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/erbyfub/episkills'>episkills</a>:</b>&nbsp;learn Python for epidemiology
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/sdwfrost/epiwidgets'>epiwidgets</a>:</b>&nbsp;A collection of htmlwidgets handy for epidemiology, etc.
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/rudeboybert/SpatialEpi'>SpatialEpi</a>:</b>&nbsp;R Package of methods and data for spatial epidemiology
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/kingaa/short-course'>short-course</a>:</b>&nbsp;Model-based Inference in Ecology and Epidemiology
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/geniusphil/2019GenomicsEpidemiologyWorkshop'>2019GenomicsEpidemiologyWorkshop</a>:</b>&nbsp;2019 Genomics Epidemiology Workshop at Academia Sinica
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/explodecomputer/WholeGenomesPractical'>WholeGenomesPractical</a>:</b>&nbsp;Whole genomes practical for the SSCM Genetic Epidemiology Short Course
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/migariane/Causal-Inference-in-Epidemiology'>Causal-Inference-in-Epidemiology</a>:</b>&nbsp;Causal Inference in Epidemiology
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/ostojanovic/BSTIM'>BSTIM</a>:</b>&nbsp;A Bayesian spatio-temporal interaction model for epidemiology.
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/joshuakuepper/Challenge-1757'>Challenge-1757</a>:</b>&nbsp;Predict the Curve Flattening
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/johannes-ko/flatten-the-curve'>flatten-the-curve</a>:</b>&nbsp;Agent based 2D virus spread simulation in Python/Qt. For playful investigation on what works best to flatten the curve of the corona virus outbreak 2020.
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/carlaiau/flatten-the-curve'>flatten-the-curve</a>:</b>&nbsp;COVID-19: By the numbers. A Website conveying the importance of early interventions in preventing disease spread. Statically generated and live at flattenthecurve.co.nz.
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/markpiercey/flatten-the-curve'>flatten-the-curve</a>:</b>&nbsp;(missing)
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/ahurford/flatten-the-curve'>flatten-the-curve</a>:</b>&nbsp;(missing)
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/DaveAckley/SPLATTheCurve'>SPLATTheCurve</a>:</b>&nbsp;Code used in the "Flatten The Curve" systems demo video
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/tinu-schneider/Flatten_the_Curve'>Flatten_the_Curve</a>:</b>&nbsp;Make the chart in Michael HÃ¶hle's blog post "Flatten the COVID-19 curve" interactive
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/RHDZMOTA/social-distancing-simulation'>social-distancing-simulation</a>:</b>&nbsp;Flatten the curve!
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/jbrowse-themes/ThemeComponent-Flat'>ThemeComponent-Flat</a>:</b>&nbsp;Flatten many curved surfaces
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/FlattenTheCurve-sk/FlattenTheCurve-sk'>FlattenTheCurve-sk</a>:</b>&nbsp;(missing)
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/markcmiller86/flatten-the-curve'>flatten-the-curve</a>:</b>&nbsp;A small site to blog about SARS-CoV-2 guidance
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/seungjaeryanlee/flatten-the-curve'>flatten-the-curve</a>:</b>&nbsp;Collection of projects and resources on COVID-19
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/DennisWeiss/flatten-the-curve'>flatten-the-curve</a>:</b>&nbsp;COVID-19 spread analysis
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/makerspacebonn/CurveFlattener'>CurveFlattener</a>:</b>&nbsp;3D-STL model for opening doors without touching the knobs. #FlattenTheCurve!
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/meflynn/flatten-curve-gif'>flatten-curve-gif</a>:</b>&nbsp;Flatten the curve gif script
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/Rayzr522/ExpFlattener'>ExpFlattener</a>:</b>&nbsp;Flattens the Minecraft EXP gain curve
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/primus852/contagion'>contagion</a>:</b>&nbsp;Different Scenarios for flattening the Curve
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/nd-rw/COVID-19-AU'>COVID-19-AU</a>:</b>&nbsp;"flattening the curve" in the context of Australia
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/Cryption/covid19-sim'>covid19-sim</a>:</b>&nbsp;simulator of virus spread / flattening the curve / herd immunity
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/JustinStitt/socialDistanceSim'>socialDistanceSim</a>:</b>&nbsp;A simulation of how Social Distancing "Flattens the Curve"
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/ArtemSokolov/covidcurve'>covidcurve</a>:</b>&nbsp;Fitting the curve that we are trying to flatten
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/Hadrien-Montanelli/covid19'>covid19</a>:</b>&nbsp;Python code - exponential spread of a disease/how to flatten the curve
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/dave-sge/-WirVsVirus'>-WirVsVirus</a>:</b>&nbsp;Herausforderung: 0038 Daten: Wie kÃ¶nnen wir Daten besser aufbereiten und nutzen? - Predict the Curve Flattening
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/estiens/slowthespread'>slowthespread</a>:</b>&nbsp;images and banners to encourage social distancing measures to flatten the curve of COVID-19/ Coronavirus
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/Corona-Killer/GutscheinGang'>GutscheinGang</a>:</b>&nbsp;A project, where vouchers can be bought to support local businesses that have big losses during the Corona crisis. Those vouchers are supposed to be redeemed after the crisis and therefore flatten the impact curve of the crisis on those businesses.
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/Xen0phobe/Gcode-Corrector-for-Laserbot'>Gcode-Corrector-for-Laserbot</a>:</b>&nbsp;Feed this program a gcode file generated with gcodetools for Inkscape. The program will remove the Z-axis and any other extraneous commands that will cause mLaser to get confused. To convert curves to straight lines I recommend using the Inkscape extension, Modify Path > Flatten Beziers. To run, create a local server and serve index.html (I recommend Live-Server).
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/tommai78101/Unity-RTS-Tweak'>Unity-RTS-Tweak</a>:</b>&nbsp;Using Unity to develop a real-time strategy game involving tweaking the game balances to reduce complexity and flatten the learning curve associated with it.
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/sfztools/sfz-flat'>sfz-flat</a>:</b>&nbsp;A command line client that "flattens" a .sfz file, removing all the hierarchies and keeping only the regions, curves and control information.
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/ljeroue/Covid-19_Trends'>Covid-19_Trends</a>:</b>&nbsp;Using Johns Hopkins Data - Look at top county and top US state death trends over time. Are we flattening the curve?
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/csamuelsm/covid19-simulations'>covid19-simulations</a>:</b>&nbsp;A simulation of the covid-19 dissemination made with Processing and inspired by the 'Why outbreaks like coronavirus spread exponentially, and how to â€œflatten the curveâ€' article on the Washington Post and written by Harry Stevens
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/mdozmorov/Immuno_notes'>Immuno_notes</a>:</b>&nbsp;Immunology-related bioinformatics data and tools
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/burkesquires/immunology-informatics'>immunology-informatics</a>:</b>&nbsp;Immunology Informatics - Big Data Analysis in Immunology - Tutorials
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/arosenfeld/immunedb'>immunedb</a>:</b>&nbsp;ImmuneDB - A system for the analysis and exploration of high-throughput adaptive immune receptor sequencing data
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/vjsingh/Virus-Game'>Virus-Game</a>:</b>&nbsp;An award-winning educational Immunology game
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/MarioniLab/ImmuneAging2017'>ImmuneAging2017</a>:</b>&nbsp;Code for Celia and Nils' immune aging project.
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/mhoore/braimmu'>braimmu</a>:</b>&nbsp;Equation-based brain immunology simulator
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/BaderLab/HumanLiver'>HumanLiver</a>:</b>&nbsp;R Data: Single cell RNA sequencing of human liver reveals distinct intrahepatic macrophage populations
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/anniez03/Immunology-VR'>Immunology-VR</a>:</b>&nbsp;Immunology VR desktop
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/herbion/immunology'>immunology</a>:</b>&nbsp;Medical system
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/SVAI/bearlyunderstanding'>bearlyunderstanding</a>:</b>&nbsp;ML based prediction of immunogenic neoantigens for immunotherapy
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/msfuji/immunoduct'>immunoduct</a>:</b>&nbsp;Pipeline for tumor immunology
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/hallamlab/utilities'>utilities</a>:</b>&nbsp;A repository of bioinformatics utilities and scripts commonly used in the Steven J. Hallam Laboratory in the Microbiology & Immunology Department of the University of British Columbia (UBC), Vancouver, British Columbia, Canada.
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/b-cell-immunology/sciReptor'>sciReptor</a>:</b>&nbsp;Single-Cell Immunoglobulin Repertoire Toolkit
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/manjunath5496/Immunology-Books'>Immunology-Books</a>:</b>&nbsp;"In whatever disease sleep is laborious, it is a deadly symptom; but if sleep does good, it is not deadly." â€• Hippocrates, The Aphorisms of Hippocrates
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/nf-core/hlatyping'>hlatyping</a>:</b>&nbsp;Precision HLA typing from next-generation sequencing data
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/VolodymyrChemodurov/ImmunologySpring'>ImmunologySpring</a>:</b>&nbsp;Immunology system
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/Transipedia/dekupl-run'>dekupl-run</a>:</b>&nbsp;Identify differentially expressed k-mers between RNA-Seq datasets
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/mer-packages/libaccounts-qt'>libaccounts-qt</a>:</b>&nbsp;libaccounts-qt packaging in Mer
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/MEEIBioinformaticsCenter/phy-mer'>phy-mer</a>:</b>&nbsp;phy-mer
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/dib-lab/2013-khmer-counting'>2013-khmer-counting</a>:</b>&nbsp;Paper code: "These are not the k-mers you are looking for: efficient online k-mer counting using a probabilistic data structure"
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/mer-packages/qtbase'>qtbase</a>:</b>&nbsp;Packaging for qtbase in Mer
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/mer-packages/qtdeclarative'>qtdeclarative</a>:</b>&nbsp;qtdeclarative packaging in Mer
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/alexforencich/verilog-mersenne'>verilog-mersenne</a>:</b>&nbsp;Verilog implementation of Mersenne Twister PRNG
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/picklejason/coronavirus-bot'>coronavirus-bot</a>:</b>&nbsp;Discord Bot for info on novel coronavirus (COVID-19)
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/YihangJoe/2019-Novel-Coronavirus'>2019-Novel-Coronavirus</a>:</b>&nbsp;ä¸€æ¬¾é è°±çš„å®æ—¶ç–«æƒ…æŸ¥è¯¢çš„APPï¼Œ2019-nCoVå…¨ä¸­å›½ç–«æƒ…åœ°å›¾å®æ—¶æ˜¾ç¤ºï¼Œå„çœå¸‚å®æ—¶ç–«æƒ…æ•°æ®æŸ¥è¯¢ï¼ŒåŒè¡Œç¨‹æŸ¥è¯¢ï¼ŒåŒºåŸŸå®šä½æŸ¥è¯¢ï¼Œé¢„é˜²æªæ–½ã€åœ¨çº¿åŒ»ç–—ç­‰ç½‘ç»œèµ„æºèšåˆå¹³å°ï¼Œå¿ å®çš„è®°å½•ä¸‹å…¨ä½“ä¸­å›½äººæ°‘åœ¨ç–«æƒ…æœŸé—´çš„ä¼—å¿—æˆåŸã€‚
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/JeroenKools/covid19'>covid19</a>:</b>&nbsp;Data visualizations of the spread of the 2019 novel Coronavirus (COVID-19), based on data from Johns Hopkins University.
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/gaospecial/NCPdashboard'>NCPdashboard</a>:</b>&nbsp;Novel Coronavirus Pneumonia (NCP-2019) Dashboard
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/CryptoKass/ncov-data'>ncov-data</a>:</b>&nbsp;2019-2020 Novel Coronavirus Data gathered from: JHU, DXY, Wikipedia and BNONews.
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/bitter-avalanche/Pandemic'>Pandemic</a>:</b>&nbsp;Pandemic is a schema for Clockwork under development by Songbird and some other freelance helpers.
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/gertvv/pandemic'>pandemic</a>:</b>&nbsp;Pandemic cooperative board game (client / server in Javascript)
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/memiah/contact-tracer'>contact-tracer</a>:</b>&nbsp;A prototype contact tracer app for COVID-19 pandemic response
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/prisoner-of-azkanban/pandemic'>pandemic</a>:</b>&nbsp;(missing)
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/Joesalmon1985/PandemicBoardGame'>PandemicBoardGame</a>:</b>&nbsp;An attempt at making a python script that lets you play the Pandemic Board Game
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/jsnider3/Pandemonium'>Pandemonium</a>:</b>&nbsp;An Analysis of Pandemic
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/whatadewitt/symmetric-pandemic'>symmetric-pandemic</a>:</b>&nbsp;JS based deck manager for symmetric pandemic games
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/anthonybishopric/pandemic-nerd-hurd'>pandemic-nerd-hurd</a>:</b>&nbsp;Tools for probability calculation in Pandemic: Legacy
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/mythmon/pl2-card-tracker'>pl2-card-tracker</a>:</b>&nbsp;Pandemic Legacy Season 2 - Card tracker
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/ZachBarbre/pandemic-georgia'>pandemic-georgia</a>:</b>&nbsp;A version of the board game Pandemic build for the backend project at DigitalCrafts
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/taura/pandemic-installer'>pandemic-installer</a>:</b>&nbsp;Automatically exported from code.google.com/p/pandemic-installer
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/Swiss-Sarge/SAR_AI'>SAR_AI</a>:</b>&nbsp;SAR AI framework for DayZ
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/casastorta/python-sar'>python-sar</a>:</b>&nbsp;Python SAR parser module
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/TAlexPerkins/sarscov2_unobserved_example'>sarscov2_unobserved_example</a>:</b>&nbsp;(missing)
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/m1awicki/mobilecoronatracker'>mobilecoronatracker</a>:</b>&nbsp;Android app for tracking SARS-CoV-2 outbreak.
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td><td></td><td></td></tr>
<tr><td><b><a href='https://github.com/wuhan2020/wuhan2020-frontend-react-app'>wuhan2020-frontend-react-app</a>:</b>&nbsp;(missing)
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td></tr>
<tr><td><b><a href='https://github.com/wuyuqiao/computationalphysics_N2013301020142'>computationalphysics_N2013301020142</a>:</b>&nbsp;Wuhan University, School of Physics, 2013 Hongyi Class, Wu Yuqiao
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td></tr>
<tr><td><b><a href='https://github.com/wuhanhelper/flutter_wuhan'>flutter_wuhan</a>:</b>&nbsp;æ­¦æ±‰ç–«æƒ…flutterç«¯
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td></tr>
<tr><td><b><a href='https://github.com/JingMatrix/university-programming'>university-programming</a>:</b>&nbsp;GitHub in WuHan University
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td></tr>
<tr><td><b><a href='https://github.com/UniqueBAT/wuhanh'>wuhanh</a>:</b>&nbsp;(missing)
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td></tr>
<tr><td><b><a href='https://github.com/LouisYZK/ShiXi_inWuhan'>ShiXi_inWuhan</a>:</b>&nbsp;18å¯’å‡äºæ­¦æ±‰å®ä¹ 
</td><td>2.4</td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td></td><td>2.4</td></tr>
</table><p>Built 2020-03-24 22:40:09.823775 <a href="results.csv" target="_blank">(CSV version for spreadsheets)</a></p></html>
